hack/release.sh

```diff
@@ -27,7 +27,7 @@ declare -A COMPONENTS
 COMPONENTS=(
   ["eventing.yaml"]="config"
   ["in-memory-channel.yaml"]="config/provisioners/in-memory-channel"
-  ["kafka-channel.yaml"]="config/provisioners/kafka"
+  ["kafka.yaml"]="config/provisioners/kafka"
```

Comment:
or, should that be:

```
["kafka.yaml"]="config/provisioners/kafka/kafka.yaml"
```

to not include the `broker` sample part ? (see: https://github.com/knative/eventing/tree/master/config/provisioners/kafka/broker)

//cc @grantr

Reply:
good - IMO the sample should not be in